bolt-sidecar/src/state/execution.rs

```diff
@@ -113,6 +117,8 @@ pub struct ExecutionState<C> {
     chain_id: u64,
     /// The maximum number of commitments per slot.
     max_commitments_per_slot: NonZero<usize>,
+    /// The maximum committed gas per slot.
+    max_committed_gas_per_slot: NonZero<u64>,
     /// The KZG settings for validating blobs.
```

Comment:
Is it better to make them into a single field - `limits`?

Reply:
yes in my opinion

Reply:
done